### PR TITLE
Improve the CONTRIBUTING/Newcomers page

### LINKED ISSUE #3393: Documentation: Newcomers/Contributing.rst
In the [Contributing guidelines](https://github.com/sunpy/sunpy/blob/master/CONTRIBUTING.rst)
- `conda` commands are not up to date (you can  activate an environment with `conda activate ...`) 
- the **hint** fork the repo and then to clone the forked repo  can be easily overlooked

### PR DESCRIPTION
**Pull request by guntbert**
### Description

- bring a `conda` command up to date
- make a valuable hint better visible

Fixes #3393

### CODE REVIEW

**Comment by guntbert** [comment]
Do I really need to add this to the changelog?

File: changelog/3393.doc.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (nabobalis) [inline changelog/3393.doc.rst]: You can delete this

File: changelog/3394.doc.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+
+- Bring the `conda` command to activate an environment up to date
+- Make the hint to fork the sunpy-repo more visible
```
Comment (nabobalis) [inline changelog/3394.doc.rst]:
```suggestion
Improved the contributing guide by updating commands and highlighting text. 
```
Just replace the whole file with that.

**Comment by nabobalis** [comment]
Conda build failure known and unrelated.

File: changelog/3393.doc.rst (hunk 1 above)
Comment (guntbert) [reply changelog/3393.doc.rst]: of course - seems to be a git error on my side :-(
I renamed the file (instead of `git mv`), committed  and pushed.

Comment (nabobalis) [reply changelog/3393.doc.rst]: That is perfectly fine.

**Comment by guntbert** [comment]
BTW: thanks for the terrific guidance (as well from the bot as from @nabobalis)

File: changelog/3394.doc.rst [hunk 3]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (nabobalis) [inline changelog/3394.doc.rst]:
```suggestion
```

File: changelog/3394.doc.rst [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+- Bring the `conda` command to activate an environment up to date
```
Comment (nabobalis) [inline changelog/3394.doc.rst]:
```suggestion
```

Comment (nabobalis) [reply changelog/3394.doc.rst]: Forgot to mention this should be deleted.

Comment (guntbert) [reply changelog/3394.doc.rst]: ah, no empty lines - makes sense
sorry to cause you so much work

**Comment by nabobalis** [comment]
Thanks @guntbert

**Pull request merged**